city_scrapers/spiders/chi_ssa_35.py

```diff
@@ -0,0 +1,147 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa35Spider(CityScrapersSpider):
+    name = "chi_ssa_35"
+    agency = "Chicago Special Service Area #35 Lincoln Ave"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.lincolnparkchamber.com/"
+        + "businesses/special-service-areas/lincoln-avenue-ssa/ssa-administration/"
+    ]
+
+    def parse(self, response):
+        content = []
+        _year = None
+        _type = ""
+        _description = ""
+
+        content_div = response.css("div.content_block.content.background_white")
+        inner_elements = content_div.css("h4, li, p")
+
+        for element in inner_elements:
+            element_content = {}
+
+            if "<h4>" in element.get():
+                # Type here could be Schedule, Agendas or Minutes
+                title = element.css("::text").get().split()
+                _type = title[-1]
+                try:
+                    _year = int(title[0])
+                except ValueError:
+                    _year = None
+
+            if "<p><em>" in element.get():
+                # The encode part here is to remove \xa0 and other characters like that
+                _description = (
+                    element.css("::text").get().encode("ascii", "ignore").decode()
+                )
+
+            if "<li>" in element.get():
+                element_content["date"] = element.css("::text").get()
+                if "href" in element.get():
+                    element_content["url"] = element.css("a::attr(href)").get()
+
+            if "<p><a" in element.get():
+                element_content["date"] = element.css("::text").get()
+                element_content["url"] = element.css("a::attr(href)").get()
+
+            element_content["description"] = _description
+            element_content["type"] = _type
+            element_content["year"] = _year
+            content.append(element_content)
+
+        self._add_year_to_date_item(content)
+        self._parse_date_to_datetime(content)
+
+        for item in content:
```

Comment:
Is there a reason for the two separate looks here? It seems like we could consolidate them and have the individual parse functions handle getting details from a parent element rather than as separate steps of a loop

Reply:
I actually don't have a reason for using two for loops. For me it was more simpler to save all data into a list and the just run the data that is into the list, but i can change that too. Anyways, the suggestion that you made about search for the elements and iterating over them was just like the way i did? Or did you have something else in mind?

Reply:
Done!